tm2/pkg/bft/abci/types/mock_application.go

```diff
@@ -0,0 +1,89 @@
+package abci
```

Comment:
Can we move this to an `internal` package if it's shared, or inside a `_test.go` file if not?

I'm usually not a huge fan of having mocks lay around like this in code, but this makes sense because I see how we can utilize it later on

Reply:
done in 404fcd59a71e7c55345608ccf2237fc24c1a0c10

I used `testing` as package name since the linter warns against importing stuff from a package named `internal`

Reply:
I could also move the `MockApplication` directly in the `consensus` package since it's only used here but I think it belongs more next the the `BaseApplication` definition